dotcom-rendering/index.d.ts

```diff
@@ -515,16 +515,16 @@ interface CAPIArticleType {
 
 type StageType = 'DEV' | 'CODE' | 'PROD';
 
-interface FrontType {
-	pressedPage: PressedPageType;
+interface FEFrontType {
```

Comment:
Nice!

Maybe add `/* FE* types are coming from Frontend */`

Reply:
Is there a reason not to call it Frontend? Just because it's easier for someone to fathom when approaching the codease. Or maybe a comment pointing to the type in that repo?

Reply:
I know it might be explanatory, but maybe worth describing DCR as well.